src/main/java/com/simanavets/tenderflexbackend/controller/LoginController.java

```diff
@@ -0,0 +1,15 @@
+package com.simanavets.tenderflexbackend.controller;
+
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/tender-flex")
```

Comment:
You should not use "tender-flex" because it relevant for all controllers. Usually you must use URL like controller name , in this case - "login"